swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+   def __init__(self, id = None, condition = 0, age = None):
+      self.id = uuid.uuid1().int if id is None else id
+      self.category = "Item"
+      self.condition = condition
+      self.age = age
+   
+   def get_category(self):
+      return self.category
+   
+   def __str__(self):
+      return f"An object of type Item with id {self.id}."
```

Comment:
You hardcode "Item" in this string. This approach limits how this function can be used by other classes which defeats the point of OOP.

What if I wanted to use this method for an instance of Decor or Electronics? 

If you invoke `get_category` you can avoid hardcoding "Item" and you open up the possibility of child classes using this function too. 

```suggestion
      return f"An object of type {self.get_category()} with id {self.id}."
```